static/app/components/eventsTable/eventsTableRow.spec.jsx

```diff
@@ -1,6 +1,6 @@
 import {render} from 'sentry-test/reactTestingLibrary';
 
-import {EventsTableRow} from 'sentry/components/eventsTable/eventsTableRow';
+import EventsTableRow from 'sentry/components/eventsTable/eventsTableRow';
```

Comment:
I would keep this and replace `withOrganization` with `useOrganization`.... but maybe you will have to fix some tests too

Reply:
yeah, it's more work :(